<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageTagDialog</class>
 <widget class="QDialog" name="ManageTagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理标签界面</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #2b2b2b;
    color: #ffffff;
}

QLabel {
    color: #ffffff;
    font-size: 12px;
}

QListWidget {
    background-color: #3c3c3c;
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 5px;
    color: #ffffff;
}

QListWidget::item {
    padding: 8px;
    border-bottom: 1px solid #555555;
}

QListWidget::item:selected {
    background-color: #0078d4;
    color: #ffffff;
}

QListWidget::item:hover {
    background-color: #4a4a4a;
}

QPushButton {
    background-color: #4a4a4a;
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 8px;
    color: #ffffff;
    min-width: 100px;
    min-height: 30px;
}

QPushButton:hover {
    background-color: #5a5a5a;
}

QPushButton:pressed {
    background-color: #666666;
}

QPushButton:disabled {
    background-color: #333333;
    color: #777777;
    border-color: #444444;
}

QFrame {
    background-color: #3c3c3c;
    border: 1px solid #555555;
    border-radius: 5px;
}

QSplitter::handle {
    background-color: #555555;
    width: 2px;
    height: 2px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">

   <item>
    <widget class="QSplitter" name="splitter_main">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="frame_tag_list1">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_tag_list1">
       <item>
        <widget class="QLabel" name="label_tag_list1">
         <property name="text">
          <string>歌曲标签列表（被转移）</string>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_tag_list1">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="toolTip">
          <string>歌曲标签列表（被转移），支持多选和拖拽调整位置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_song_list">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_song_list">
       <item>
        <widget class="QLabel" name="label_song_list">
         <property name="text">
          <string>歌曲列表</string>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_song_list">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="toolTip">
          <string>显示选中标签下的歌曲</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_tag_list2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_tag_list2">
       <item>
        <widget class="QLabel" name="label_tag_list2">
         <property name="text">
          <string>歌曲标签列表（转移到）</string>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_tag_list2">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="toolTip">
          <string>歌曲标签列表（转移到），支持多选和拖拽调整位置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_operations">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_operations">
       <item>
        <widget class="QLabel" name="label_operations">
         <property name="text">
          <string>操作</string>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_copy_transfer">
         <property name="text">
          <string>复制转移</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/new/prefix1/images/copyIcon.png</normaloff>:/new/prefix1/images/copyIcon.png</iconset>
         </property>
         <property name="toolTip">
          <string>复制选中的歌曲到目标标签</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_move_transfer">
         <property name="text">
          <string>移动转移</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/new/prefix1/images/moveIcon.png</normaloff>:/new/prefix1/images/moveIcon.png</iconset>
         </property>
         <property name="toolTip">
          <string>移动选中的歌曲到目标标签</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_undo">
         <property name="text">
          <string>撤回</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/new/prefix1/images/undoIcon.png</normaloff>:/new/prefix1/images/undoIcon.png</iconset>
         </property>
         <property name="toolTip">
          <string>撤回上一次的操作</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_select_all">
         <property name="text">
          <string>全选歌曲</string>
         </property>
         <property name="toolTip">
          <string>选中所有歌曲</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_deselect_all">
         <property name="text">
          <string>取消选中</string>
         </property>
         <property name="toolTip">
          <string>取消选中所有歌曲</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_operations">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_dialog_buttons">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_dialog_buttons">
      <item>
       <spacer name="horizontalSpacer_left">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit_discard">
        <property name="text">
         <string>退出不保存</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/new/prefix1/images/exit-discardChangesIcon.png</normaloff>:/new/prefix1/images/exit-discardChangesIcon.png</iconset>
        </property>
        <property name="toolTip">
         <string>退出对话框并丢弃所有更改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit_save">
        <property name="text">
         <string>退出并保存</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/new/prefix1/images/exit-SaveIcon.png</normaloff>:/new/prefix1/images/exit-SaveIcon.png</iconset>
        </property>
        <property name="toolTip">
         <string>保存所有更改并退出对话框</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_right">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_exit_discard</sender>
   <signal>clicked()</signal>
   <receiver>ManageTagDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_exit_save</sender>
   <signal>clicked()</signal>
   <receiver>ManageTagDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
